scripts/high_d_phaseC.py

```
@@ -32,18 +42,33 @@
-    (probability and kick-direction source)."""
+    """Arm -> extra CLI args. Phase A is shared; arms differ only in the collective phase
+    (low-mode kick source, or the category-tied move)."""
     return {
         "base": list(PHASE_A),
         "lm_pca": PHASE_A + ["--lowmode-prob", "0.5"] + _WARMUP + ["--lowmode-kick-dir", "pca"],
         "lm_enm": PHASE_A + ["--lowmode-prob", "0.5"] + _WARMUP + ["--lowmode-kick-dir", "enm"],
         "lmonly_enm": PHASE_A + ["--lowmode-prob", "1.0"] + _WARMUP + ["--lowmode-kick-dir", "enm"],
+        # Category-tied move in place of the PCA kick; prior loaded for grouping only
+        # (--prior-exponent 0 turns off PiBO acquisition steering).
+        "cat": PHASE_A + ["--category-prob", "0.5"] + _CAT_WARMUP
+        + ["--priors", PRIORS, "--prior-exponent", "0"],
```

Ruff (0.15.20)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)